TheMovieDb/MovieCell.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  MovieCell.swift
+//  TheMovieDb
+//
+//  Created by Antonio Hernandez Ambrocio on 05/11/21.
+//
+
+import Foundation
+import UIKit
+
+class MovieCell: UICollectionViewCell {
+    static let identifier = "MovieCell"
```

Comment:
Take a look at this https://github.com/AliSoftware/Reusable
you can implement it and then use a identifier automatically added